### DISCUSSION TITLE
How do i get a mushroom

### DISCUSSION THREAD

**Discussion by Begodden** [GalaxyOfZero/Skyfriends#26] [Q&A] [answered]
i have been racking my head for ages on hot to get a mushroom for the rune of earth i have tried getting a mooshroom to spawn with no luck

**Comment by GalaxyOfZero** [comment, marked as answer]
put a Ancient Spores in Witch water and your get a brown mushroom block with luck your get brown mushroom. if your need Slime put a Bohn Mehl or Brown mushroom in witch Water

**Reply by GalaxyOfZero** [reply]
if your have any questions Join the SkyFriends Discore!!!

**Reply by Begodden** [reply]
spot on thank you
